hubble-be/pom.xml

```diff
@@ -67,10 +67,37 @@
             <version>2.8.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.baidu.hugegraph</groupId>
+            <artifactId>hugegraph-common</artifactId>
```

Comment:
TODO: remove after upgrading hugegraph-client v1.8.3

Reply:
yes, I rember it